replapi_it/classes/post.py

```diff
@@ -0,0 +1,36 @@
+from .base import BaseClass
+
+
+class Post(BaseClass):
+  def __init__(self, id: int, **kwargs):
+    self.vars = {**{"id": id}, **kwargs}
+    self._query_args = {"$id": "Int!"}
+    self._types = {
+      "post(id: $id)": [
+        "id",
```

Comment:
There are much more query types, right?

Reply:
And is that the same for `user.py`?

Reply:
The type fields can be found [here](https://github.com/ReplAPI-it/Replit-API-Schema/tree/master/graphql/properties). As for the types, ask @RayhanADev because I have no clue. (You could also look through the [source code or something](https://github.com/ReplAPI-it/ReplAPI.it-NodeJS/tree/master/cjs/src/classes).)

Reply:
@JBYT27 yeah there are, this is fine for right now since I don't have all the docs ready but there are quite a few more

Reply:
Got it, thanks!